Bootcamp2015.AmazingRace.BackgroundTask/BTask.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Bootcamp2015.AmazingRace.Base;
+using Bootcamp2015.AmazingRace.Base.Helpers;
+using Microsoft.WindowsAzure.MobileServices;
+using Windows.ApplicationModel.Background;
+
+namespace Bootcamp2015.AmazingRace.BackgroundTask
+{
+    public sealed class BTask : IBackgroundTask
+    {
+        public void Run(IBackgroundTaskInstance taskInstance)
+        {
+            var deferral = taskInstance.GetDeferral();
+
+            var credentials = PasswordVaultHelper.GetPasswordCredential();
+            var user = PasswordVaultHelper.GetUser();
+            var client = new MobileServiceClient(
+                Connections.MobileServicesUri, 
+                Connections.MobileServicesAppKey
+            );
+            client.CurrentUser = user;
+
+            LocationHelper.Start();
```

Comment:
This might not work as you intended since it's relying on the location change. We should force an update of the location here since the background task has a limited amount of cpu time.